CONTRIBUTING.md

```diff
@@ -24,6 +24,7 @@ The submitted code should adhere to the following:
 
 ### Do's
 
+* Keep pull requests focused on addressing one issue or feature. If multiple separate issues are fixed by a pull request, prefer commiting the fixes under multiple commits, one for each issue.
 * For every pull request, make sure all commits are consolidated into one commit before
```

Comment:
s/all commits are consolidated into one commit/all commits for a related issue or feature are consolidated into a single commit/